src/Sentry.Unity.Editor/ConfigurationWindow/DebugSymbolsTab.cs

```diff
@@ -21,6 +21,10 @@ internal static void Display(SentryCliOptions cliOptions)
                 new GUIContent("Upload Sources", "Whether your source code should be uploaded to Sentry, so the stack trace in Sentry has the relevant code next to it."),
                 cliOptions.UploadSources);
 
+            cliOptions.IgnoreCliErrors = EditorGUILayout.Toggle(
+                new GUIContent("Ignore CLI Errors", "Whether to ignore the Sentry CLI errors during the build. If this is ON, the errors will be logged, but the build will succeed."),
```

Comment:
```suggestion
                new GUIContent("Ignore Symbol Upload Errors", "Whether to ignore the Sentry CLI errors during the build. If this is ON, the errors will be logged, but the build will succeed."),
```

I'm not sure if there's enough space for that "name" but otherwise we can drop the "symbol".